schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
```

Comment:
AFAIK, that depends on user rights, so they'd need to be logged in. So I'd say *no, historic records for such a page should not be publicly available*.